idearium-lib/index.js

```diff
@@ -5,7 +5,8 @@ module.exports = {
     // Packages here.
     mq: require('./lib/mq'),
     logs: require('./lib/logs'),
-    utils: require('./lib/utils'),
+    util: require('./lib/util'),
+    utils: require('./lib/utils'), // This should eventually be merged into the util module above.
```

Comment:
I think we should do this now. Introducing `util` while having `utils` is going to be very confusing. What we should do is:

- keep the function in `utils` but wrap it with https://nodejs.org/api/util.html#util_util_deprecate_function_string
- copy the one function across to `util`.